src/code/fv-components/fv-gene-label.js

```diff
@@ -13,6 +13,10 @@ const Form = React.createClass({
     // render :: a -> ReactElement
     render: function(){
         return <SimpleSelect defaultValue={this.props.defaultOption} onValueChange={this.props.handleChange} options = {this.props.options} hideResetButton={true} editable={false} disabled={false}></SimpleSelect>;
+    },
+
+    componentDidMount: function() {
+      document.querySelectorAll("input").forEach(input => input.disabled = true);
```

Comment:
Yes, as you suggested, I did find a better library after the fact, where "searchable" is a configurable parameter. Something something hindsight...

https://jedwatson.github.io/react-select/

Let me give that snippet a try!